src/lib/core/datetime/native-date-adapter.spec.ts

```diff
@@ -0,0 +1,242 @@
+import {NativeDateAdapter} from './native-date-adapter';
+
+
```

Comment:
In the material1 tests, I used constants for month indices to make the tests easier to read:
```js
  // When constructing a Date, the month is zero-based. This can be confusing, since people are
  // used to seeing them one-based. So we create these aliases to make reading the tests easier.
  var JAN = 0, FEB = 1, MAR = 2, APR = 3, MAY = 4, JUN = 5, JUL = 6, AUG = 7, SEP = 8, OCT = 9,
      NOV = 10, DEC = 11;
```

Then your tests look like 
```ts
expect(adapter.getYear(new Date(2017, JAN, 1))).toBe(2017);
```
Make a similar change here?